From a0565c0e352725982ecdb6caec6c2a541a8f9329 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Tue, 17 Jul 2012 16:52:20 +0200
Subject: [PATCH] latex2html fix defined 20 of Debian series 7

latex2html with current perl gives warnings about the deprecation of
"defined(%hash)". This patches removes them by just removing the resp.
keyword.
Author: Roland Stigge <stigge@antcom.de>
---
 latex2html.pin |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/latex2html.pin b/latex2html.pin
index fdf68c2..d670c0d 100644
--- a/latex2html.pin
+++ b/latex2html.pin
@@ -532,7 +532,7 @@ $CHARSET = $charset || 'iso-8859-1';
 #
 # If possible, use icons of the same type as generated images
 #
-if ($IMAGE_TYPE && defined %{"icons_$IMAGE_TYPE"}) {
+if ($IMAGE_TYPE && %{"icons_$IMAGE_TYPE"}) {
     %icons = %{"icons_$IMAGE_TYPE"};
 }
 
@@ -2112,7 +2112,7 @@ sub convert_iso_latin_chars {
 		"\nCould not find translation function for $default_language.\n\n")
 	    }
 	);
-	if ($USE_UTF ||(!$NO_UTF &&(defined %unicode_table)&&length(%unicode_table)>2)) {
+	if ($USE_UTF ||(!$NO_UTF &&(%unicode_table)&&length(%unicode_table)>2)) {
 	    &convert_to_unicode($_)};
     }
     $_ = join('', @case_processed, $_); undef(@case_processed);
@@ -9117,7 +9117,7 @@ sub real_replace_strange_accents {
     my ($charset) = "${CHARSET}_character_map_inv";
     $charset =~ s/-/_/g;
     # convert upper 8-bit characters
-    if (defined %$charset &&($CHARSET =~ /8859[_\-]1$/)) {
+    if (%$charset &&($CHARSET =~ /8859[_\-]1$/)) {
 	s/([\200-\377])/
 	    $tmp = $$charset{'&#'.ord($1).';'};
 	    &mark_string($tmp) if ($tmp =~ m!\{!);
-- 
1.7.10.3

